src/format/KeePass2Repair.cpp

```diff
@@ -29,8 +29,14 @@ KeePass2Repair::KeePass2Repair()
 {
 }
 
+KeePass2Repair::~KeePass2Repair()
+{
+    delete m_db;
```

Comment:
Make `m_db` a `QScopedPointer` and remove this code.

Reply:
done